pages/your_domain/human_data.md

```diff
@@ -10,7 +10,7 @@ training:
   - name: Training in TeSS
     registry: TeSS
     url: https://tess.elixir-europe.org/search?q=sensitive%20human%20data
-  - name: RDMbites for sensitive data
+  - name: RDMbites about data sensitivity
```

Comment:
It was decided to work on making sure that RDMbites result on TeSS search, to avoid direct link to single entry in TeSS. It could be adapted later.